cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/constraint/LastRunConstraint.java

```diff
@@ -30,19 +30,19 @@
  */
 public class LastRunConstraint extends ProtoConstraint.LastRunConstraint implements CheckableConstraint {
 
-  public LastRunConstraint(long millisSinceLastRun) {
-    super(millisSinceLastRun);
+  public LastRunConstraint(long secsSinceLastRun) {
```

Comment:
How about updating the delay and lastRun APIs to be similar to `setTimout` parameters?
https://github.com/caskdata/cdap/blob/63a8417827f466bc6904eeae154c3b5a2ab23f95/cdap-api/src/main/java/co/cask/cdap/api/schedule/ScheduleBuilder.java#L63-L63